db/migrate/20161014133811_move_content_view_version_description_to_histories.rb

```diff
@@ -1,12 +1,13 @@
 class MoveContentViewVersionDescriptionToHistories < ActiveRecord::Migration
-  class Katello::ContentViewVersion < ApplicationRecord
+  class Katello::FakeContentViewVersion < ApplicationRecord
```

Comment:
Also if you felt namespacing was safer I 'd recommend giving it a "Katello::Migration" or some such unique name